tapir/coop/forms.py

```diff
@@ -247,3 +251,110 @@ class Meta:
         }
 
     share_owner = ShareOwnerChoiceField()
+
+
+class MembershipResignationForm(forms.ModelForm):
+    already_resigned = MembershipResignation.objects.values("share_owner")
+    in_three_years = _("Coop buys back share(s)")
+    cancellation_reason = forms.CharField(
+        max_length=1000,
+        widget=forms.Textarea(
+            attrs={"rows": 2, "placeholder": _("Please not more than 1000 characters.")}
+        ),
+    )
+    coop_buys_shares_back = forms.BooleanField(label=in_three_years, required=False)
+    share_owner = ShareOwnerChoiceField()
+    willing_to_gift_shares_to_coop = forms.BooleanField(
+        label="Willing to gift shares to coop", required=False
```

Comment:
It could be nicer to define `verbose_name` and `help_text` in the model's field rather than the form field, then the name and help text would get reused if we ever do other forms. Also, you can then remove this class variable entirely (I think?) sind the Meta.fields already has all the required information.

Reply:
Like this?
`willing_to_gift_shares_to_coop = models.BooleanField(
        "Willing to gift shares to coop", 
        help_text = "Willing to gift shares to coop", 
        required = False,
        )`